scipy/interpolate/interpolate.py

```diff
@@ -623,17 +623,16 @@ def _check_bounds(self, x_new):
 
 
 class _PPolyBase(object):
-    """
-    Base class for piecewise polynomials.
-    """
+    """Base class for piecewise polynomials."""
     __slots__ = ('c', 'x', 'extrapolate', 'axis')
 
-    def __init__(self, c, x, extrapolate=None, axis=0):
```

Comment:
I thought that it can have some reasons, it makes some sense, but I don't completely get it. Anyway I'm changing it back.